agent/engine/task_manager.go

```diff
@@ -26,6 +26,7 @@ import (
 
 const (
 	steadyStateTaskVerifyInterval         = 5 * time.Minute
+	executionStoppedAtSubmitDelay         = 12 * time.Hour
```

Comment:
This constant needs a comment.  I don't know what it represents or why it's set to 12 hours.

Reply:
will add comment, actually this is the maximum delay can be accepted by backend.